app/models/course/assessment/question/programming_test_case.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 class Course::Assessment::Question::ProgrammingTestCase < ActiveRecord::Base
+  acts_as_duplicable
```

Comment:
@fonglh  I think the declaration of `acts_as_duplicable` is not useful anymore, because the exception will always be  raised if `initialize_duplicate` is not defined.  Or do you have any plan in future to use this /